soc/arm/microchip_mec/mec172x/reg/mec172x_gpio.h

```diff
@@ -427,7 +454,7 @@ struct gpio_ctrl_regs {
 	volatile uint32_t CTRL_0036;
 	uint32_t RSVD3[1];
 	volatile uint32_t CTRL_0040;
-	uint32_t RSVD4[1];
+	volatile uint32_t CTRL_0041;
```

Comment:
Any specific reason for the exclusion of below members?
CTRL_0001
CTRL_0005
CTRL_0006
CTRL_0230 to CTRL_0236
CTRL_0251 and CTRL_0252